### PR TITLE
Replace default example initialization messages

### LINKED ISSUE #148: SimpleKernelTimer: strange initialization message (`Example Library Initialized`)
https://github.com/kokkos/kokkos-tools/blob/c901382c4c76b108e4e6d190e9236848dc764526/profiling/simple-kernel-timer/kp_kernel_timer.cpp#L120

It seems this name is not correct. It should say `SimpleKernelTimer loaded` or something like that. I suppose something along the following:
https://github.com/kokkos/kokkos-tools/blob/c901382c4c76b108e4e6d190e9236848dc764526/profiling/memory-events/kp_memory_events.cpp#L85

### PR DESCRIPTION
**Pull request by vlkale**
Fix #148

### CODE REVIEW

**Review by masterleinad — APPROVED** [review]

**Comment by vlkale** [comment]
Better initialization messages in kp_kernel_timer.cpp  kp_kernel_logger.cpp. 

The change is very straightforward and has been gone over with @dalg24.

**Review by dalg24 — APPROVED** [review]
Looks good.  Did you mean to leave the to high-water-memory tools out?
```
debugging/kernel-logger/kp_kernel_logger.cpp:82:        printf("KokkosP: Example Library Initialized (sequence is %d, version: %llu)\n", loadSeq, interfaceVer);
profiling/memory-hwm-mpi/kp_hwm_mpi.cpp:74:    printf("KokkosP: Example Library Initialized (sequence is %d, version: %" PRIu64 ")\n", loadSeq, interfaceVer);
profiling/memory-hwm/kp_hwm.cpp:64:     printf("KokkosP: Example Library Initialized (sequence is %d, version: %llu)\n", loadSeq, interfaceVer);
profiling/simple-kernel-timer/kp_kernel_timer.cpp:120:  printf("KokkosP: Example Library Initialized (sequence is %d, version: %llu)\n", loadSeq, interfaceVer);

**Comment by vlkale** [comment]
@dalg24 Thanks for catching. Will fix.

**Comment by vlkale** [comment]
The High Water Mark file is fixed now.

@dalg24 @masterleinad

**Comment by dalg24** [comment]
You missed `profiling/memory-hwm/kp_hwm.cpp:64`

File: profiling/memory-hwm/kp_hwm.cpp [hunk 1]
```diff
@@ -61,7 +61,7 @@ extern "C" void kokkosp_init_library(const int loadSeq,
 	const uint32_t devInfoCount,
 	void* deviceInfo) {
 
-	printf("KokkosP: Example Library Initialized (sequence is %d, version: %llu)\n", loadSeq, interfaceVer);
+	printf("KokkosP: Kokkos High Water Mark Library Initialized (sequence is %d, version: %llu)\n", loadSeq, interfaceVer);
```
Comment (dalg24) [inline profiling/memory-hwm/kp_hwm.cpp]:
```suggestion
	printf("KokkosP: High Water Mark Library Initialized (sequence is %d, version: %llu)\n", loadSeq, interfaceVer);
```

**Review by dalg24 — APPROVED** [review]
Apply the suggestion and do squash merge

**Pull request merged**